docs/integration_guide/rest_api/authentication.rst

```diff
@@ -0,0 +1,154 @@
+Authentication
+==============
+
+All calls to REST API endpoints must be authenticated as a particular User.
+
+.. tip::
+  The password for the "admin" user can be set using two methods.
+
+      ``pulp-manager reset-admin-password``
+
+  The above command prompts the user to enter a new password for "admin" user.
+
+      ``pulp-manager reset-admin-password --random``
+
+  The above command generates a random password for "admin" user and prints it to the screen.
+
+.. tip::
+  If you are using django rest framework browsable API these browser addons may come handy:
+
+  * Chrome `ModHeader <https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj>`_
+  * Firefox `Modify Headers <https://addons.mozilla.org/cs/firefox/addon/modify-headers/>`_
+
+Basic Authentication
+--------------------
+
+Any call to the REST API may use
+`HTTP basic authentication <http://tools.ietf.org/html/rfc1945#section-11.1>`_ to provide
+a username and password.
+
+JWT Authentication
+------------------
+
+Alternatively you can use `JSON Web Tokens authentication <https://tools.ietf.org/html/rfc7519>`_.
+
+Token Structure
+^^^^^^^^^^^^^^^
+
+The structure of the token consists of ``username`` and ``exp``. The tokens are signed by each
+user's individual secret by HMAC SHA-256 ("HS256") algorithm.
+
+Example:
+::
+    {
+      "username": "admin",
+      "exp": 1501172472
+    }
+
+Obtaining token from server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can "login" to Pulp by supplying your credentials in POST request and obtain a JWT token
+with expiration time set by ``JWT_AUTH.JWT_EXPIRATION_DELTA``
+
+
+ * **method:** ``post``
+ * **path:** ``api/v3/jwt/``
+ * response list:
+    * **code:** ``200`` - credentials were accepted
+    * **code:** ``400`` - credentials are wrong
+
+ **Sample POST request:**
+ ::
+  {
+    "username": "admin",
+    "password": "admin_password"
+  }
+
+
+ **Sample 200 Response Body:**
+ ::
+    {
+        "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiZXhwIjoxNTAyMzgzMDExfQ.3ZpcclxV6hN8ui2HUbwXLJsHl2lhesiCPeDVV2GIbJg"
+    }
+
+Using a token
+^^^^^^^^^^^^^
+
+For using JWT tokens you have to set ``Authorization`` header as follows:
+::
+  Authorization: JWT eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiZXhwIjoxNTAyMzgzMDExfQ.3ZpcclxV6hN8ui2HUbwXLJsHl2lhesiCPeDVV2GIbJg
```

Comment:
Why are you using `JWT` here? On [the jwt.io site](https://jwt.io/introduction/#how-do-json-web-tokens-work-) and elsewhere I see `Bearer` being used.

Reply:
@daviddavis It's the default value from the library they decided to use `JWT` instead of `Bearer`, which they originally used, to maintain compatibility with `django-oauth2-provider` whihc also uses it. Source: https://github.com/GetBlimp/django-rest-framework-jwt/pull/4